Projects/09_EDA_Human_Body_Temperature/Human_Body_Temp_final.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import kstest, normaltest, ttest_1samp, wilcoxon, ranksums, ttest_ind
+import matplotlib.pyplot as plt
+
+def explain_test(pval,test):
+    if pval > 0.05:
+        print('The '+ test + ' indicates that the data is NOT normally distirbuted')
+    else:
+        print('The '+ test + ' indicates that the data IS normally distirbuted')
+
+def norm_tests(xdata):
+    _, p =kstest(xdata, 'norm')
+    explain_test(p,'Kolmogorov-Smirnov test')
+    print(40*'==')
+    _, p =normaltest(xdata)
```

Comment:
`print(40*'==')` should be `print(40 * '==')`
`_, p =normaltest` should be `_, p = normaltest`